extensions/amazon-sales/release-notes/index.md

```diff
@@ -0,0 +1,44 @@
+---
+group: extensions
+title: Amazon Sales Channel Release Notes
+---
+
+*Release notes published April 2019.*
+
+**Amazon Sales Channel** is now available with Magento 2.2.x and 2.3.x. Merchants can use Amazon Sales Channel to integrate with an Amazon Seller account to post and sell Magento products on Amazon.
+
+See the following documentation:
+
+- [Amazon Sales Channel](https://docs.magento.com/m2/ce/user_guide/sales-channels/amazon/amazon-sales-channel.html) for merchant information and instructions
+- [Amazon Sales Channel install]({{site.baseurl}}/extensions/amazon-sales/) for installation and API key information
+- [Amazon Sales Channel Marketplace page](http://marketplace.magento.com/magento-module-amazon.html)
+
+The release notes include:
+
+-   {:.new}New features
+-   {:.fix}Fixes and improvements
+-   {:.bug}Known issues
+
+### v1.0.0
+
+Amazon Sales Channel 1.0.0 is compatible with version 2.2.4+ and 2.3.x of {{site.data.var.ce}}, {{site.data.var.ee}}, and {{site.data.var.ece}}.
+
+- {:.new} **Simplified Onboarding and Maintenance**: Add and integrate with your Amazon Seller account through a step-by-step process with detailed instructions available through the Magento Admin. Maintain your stores, accounts, and listed products through one dashboard.
+
+- {:.new} **Multiple Account Support**: Manage and monitor multiple Amazon brands and marketplace regions through the Magento Admin.
+
+- {:.new} **Intelligent Pricing**: Set automated repricing rules to increase your chances for the coveted Buy Box. Set prices to dynamically adjust to the current Buy Box price, or lowest competitor pricing. Set limits to repricing to protect your margin.
+
+- {:.new} **Listing Management**: Automate product listings and sync your Magento catalog to the Amazon Marketplace using listing rules. Add specific overrides to finely control your offerings. Monitor and manage all your listings directly from the Magento Admin.
+
+- {:.new} **Consistent Inventory Management**: Keep your Magento and Amazon inventory quantities in constant synchronization.
+
+- {:.new} **Order and Fulfillment Management**: Track Amazon orders through the dashboard, with seamless communication and inventory updates. Complete and track order shipments as fulfilled by Amazon, merchant fulfilled, or a mix of methods.
+
+-   {:.bug} Updates for product quantity may take ~2 hours to sync.
```

Comment:
The descriptions make it sound like these are known issues that haven't been fixed. If so, change the description of the bug icon above. If not, I think they should be worded like they've been fixed. However, as an inaugural release, I'm not sure why we would list "bug" fixes.

Reply:
Understood, this was requested for the prerelease. Will look them over and consider some rewording. Thanks!